client/src/components/CategoryShow.js

```diff
@@ -45,6 +108,9 @@ const CategoryShow = (props) => {
       <div className="list-container">
         <div className="column-grid">{pastasList}</div>
       </div>
+      <div>
+        {showForm()}
```

Comment:
I believe in react when we want to conditionally render something, we can set properties to some variable, as opposed to invoking some function.

https://reactjs.org/docs/conditional-rendering.html

Very similar to what you have here,

```js
let showForm = <h3>You must be logged in to submit a pasta!</h3>
if (currentUser) {
  showForm = ...
}

...
return (
  ...
  <div>{showForm}</div>
  ...
)
```